salt/utils/jinja.py

```diff
@@ -48,6 +54,15 @@ class OrderedDictDumper(yaml.Dumper):  # pylint: disable=W0232
                      Dumper=OrderedDictDumper)
 
 
+def jinja_filter(name=None):
```

Comment:
Should this decorator function perhaps be in the `salt.utils.decorators` module, rather than `salt.utils.jinja`?